test/router/lifecycle-events.spec.html

```diff
@@ -50,6 +50,26 @@
     describe('Vaadin.Router lifecycle events', function() {
       const {verifyActiveRoutes, onAfterEnterAction, onBeforeEnterAction, onBeforeLeaveAction} = VaadinTestNamespace;
 
+      const verifyCallbacks = expectedCallbacks => {
+        expect(callbacksLog).to.be.an('array');
+        expect(expectedCallbacks).to.be.an('array');
+
+        try {
+          expect(callbacksLog).to.deep.equal(expectedCallbacks);
+        } catch (e) {
+          // eslint-disable-next-line no-console
```

Comment:
instead of repeating yourself, I'd recommend you to contribute to https://github.com/vaadin/eslint-config-vaadin and add `'table'` to https://github.com/vaadin/eslint-config-vaadin/blob/master/index.js#L27